tilesets/TwilioQuestObjects.json

```diff
@@ -1379,7 +1373,6 @@
          "objectgroup":
             {
              "draworder":"index",
-             "id":2,
```

Comment:
These `ids` all got removed. I suspect this is part of the tileset version migration from 1.6 to 1.8.

I don't think we rely on these values, but we should definitely note this could potentially cause an issue!